index.html

```diff
@@ -108,21 +109,21 @@ <h2>Who is welcome?</h2>
 			<div class="steps__inner">
 				<div class="step">
 					<div class="step__media">
-						<img src="./images/undraw_developer_activity.svg" class="step__image">
+						<img src="./images/undraw_developer_activity.svg" class="step__image" alt="">
```

Comment:
💩

Reply:
They keep the screen readers from reading the src tags, which I believe is what they do by default